cmd/storage-provisioner/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"k8s.io/minikube/cmd/localkube/cmd"
+	"sync"
+)
+
+func main() {
+	localkubeServer := cmd.NewLocalkubeServer()
+	storageProvisionerServer := localkubeServer.NewStorageProvisionerServer()
+
+	var wg sync.WaitGroup
+	wg.Add(2)
+	go func() {
+		defer wg.Done()
+		storageProvisionerServer.Start()
```

Comment:
You can just call this directly, no need to put it in a waitgroup

Reply:
So I tried that initially, but it didn't work. I looked it up and I think the main program ends before the server has a chance to start?

Reply:
I think thats due to a few things i mentioned above.  
Once you do that, then you can see that its actually just exiting since there are errors

```
$ ./main --v 10 --logtostderr
F1030 14:08:28.887110  152274 main.go:30] unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
```